src/libmeasurement_kit/dns/system_resolver.hpp

```diff
@@ -0,0 +1,146 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <measurement_kit/dns.hpp>
+#include <measurement_kit/portable/sys/socket.h>
+#include <measurement_kit/portable/sys/types.h>
+#include <measurement_kit/portable/netdb.h>
```

Comment:
These three files should not be here. They should be included by `measurement_kit/common/aaa_base.hpp`